dokka-subprojects/core/src/main/kotlin/org/jetbrains/dokka/model/Documentable.kt

```diff
@@ -163,12 +168,13 @@ public data class DClass(
     override val modifier: SourceSetDependent<Modifier>,
     override val sourceSets: Set<DokkaSourceSet>,
     override val isExpectActual: Boolean,
-    override val extra: PropertyContainer<DClass> = PropertyContainer.empty()
+    override val extra: PropertyContainer<DClass> = PropertyContainer.empty(),
+    override val typealiases: List<DTypeAlias> = emptyList(),
```

Comment:
Please add a deprecated constructor and copy it as with `contextParameters` in `DFunction` to maintain the ABI.